content/applications/productivity/iot/devices/scale.rst

```diff
@@ -0,0 +1,54 @@
+===============
+Connect a Scale
+===============
+
+When using your *IoT Box* in Odoo, you could need to use a scale. Doing so is easy and convenient as
+it can be done in a few steps. Then, you can use it in your *Point of Sale app* to weigh your
+products, which is helpful if their price is based on it.
+
+Connection
+==========
+
+To connect the scale to the *IoT Box*, you just have to combine the two by cable.
+
+.. note::
+        In some cases, a serial to USB adapter may be needed.
+
+If your scale is a `Supported devices <https://www.odoo.com/page/iot-hardware>`__, there is no
+need to set up anything because it will be automatically detected as soon as it is connected.
+
+.. image:: scale/choix-iot.png
+   :align: center
+   :alt: IOT box auto detection.
+
+You may need to restart the box and download your scales’ drivers from the box in some cases. To do
+so, go to the *IoT Box Home Page* and click on *drivers list*. Then, click on load drivers.
+
+.. image:: scale/driver-list.png
+   :align: center
+   :alt: View of the IoT box settings and driver list.
+
+Use a Scale in Point of Sale
+============================
+
+To use the scale in your *Point of Sale app*, go to the *Point of Sale* menu, under *Configuration*
+and open a *Point of Sale* in the list. Once opened, edit it and check the *IoT Box* feature.
+
+.. image:: scale/iot-box-pos.png
+   :align: center
+   :alt: View of the IoT box feature inside of the PoS settings.
+
+Now, choose the *IoT Box* in the dropdown menu and check the *Electronic Scale* option. Then, you
+hit save.
+
+.. image:: scale/electronic-scale-feature.png
+   :align: center
+   :alt: List of the external tools that can be used with PoS and the IoT box.
+
+The scale is now available in all your *PoS* sessions. Then, if a product has a price per weight
+set, clicking on it on the *PoS* screen will open the scale screen, where the cashier can weigh the
+product and add the correct price to the cart.
```

Comment:
nitpicking:
I know it's obvious that it's only the sessions of the configured PoS but you wxrite "all" pos sessions.

+

```suggestion
The scale is now available in all your *PoS* sessions. Then, if a product has a price per weight
set, clicking on it on the *PoS* screen opens the scale screen, where the cashier can weigh the
product and add the correct price to the cart.
```